RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.10.0
+------
+* Adding a block from the post title now shows the add block here indicator.
+* Deselect post title any time a block is added
+* Fix merging of empty paragraph blocks when added from a post's title
```

Comment:
@tug In light of your comment (https://github.com/WordPress/gutenberg/pull/16677#issuecomment-515484612) that we may not want this change 
 and since we're likely going to lose it with your imminent PR merge, we might want to pull this line out of the release notes. Alternatively, I could revert the entire change from https://github.com/wordpress-mobile/gutenberg-mobile/pull/1227. What do you think?

Reply:
If you think we can easily revert it, let's do this. Otherwise keeping it shouldn't impact the user experience too much I think

Reply:
Reverted.